x-pack/plugins/ml/public/application/components/anomalies_table/anomaly_details.js

```diff
@@ -279,7 +279,7 @@ export class AnomalyDetails extends Component {
           ),
         },
         {
-          id: 'Category examples',
+          id: 'categoryExamples',
```

Comment:
```suggestion
          id: 'category-examples',
```

Reply:
Updated in 31f72d501e6c25d8a4a6b5e0375d462c03d8e1c9